### ISSUE TITLE
Not really an issue, I noticed there wasn't a guide for compiling/building melonDS on Fedora Linux but I've managed to do it so I'm sharing the libs here

### ISSUE THREAD

**Issue by diogofd8** [melonDS-emu/melonDS#1692]
`sudo dnf install git cmake extra-cmake-modules libcurl-devel libpcap-devel SDL2-devel qt5-qtbase-devel qt5-qtbase-private-devel qt5-qtmultimedia-devel libslirp-devel libarchive-devel wayland-devel libzstd-devel`

Hopefully this ends up helping someone.

**Comment by patataofcourse** [comment]
you might be better off PRing this to the README, i don't think a lot of people are going to look at this

**Comment by v-fox** [comment]
Yep. Every distro has its own convention on packaging. When you do packaging or local compilation yourself, you're expected to learn them from distro's manuals and then figure out such things on the fly. So, there is no reason to reiterate general practices.
